code/modules/mob/living/carbon/human/species.dm

```diff
@@ -795,6 +795,11 @@
 
 			if(istype(J) && J.allow_thrust(0.01, H))
 				. -= 2
+			else
+				var/obj/item/organ/internal/cyberimp/chest/thrusters/T = H.getorganslot("thrusters")
+				if(istype(T) && T.allow_thrust(0.01, H))
+					. -= 2
+
```

Comment:
Ok, testing shows that Xenos keep their movespeed in space, and are able to fly around normally, so there's no issue there. Monkeys do lack a speedboost, however this already existed with them wearing jetpacks normally. Monkeying also seems to remove implants.